my-code/movie-search/src/Containers/MovieDetails/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { fetchMovie } from '../../Actions';
+
+class MovieDetails extends Component {
+  componentDidMount() {
+    const { imdbID } = this.props.match.params;
+    this.props.fetchMovie(imdbID);
+
+    console.log(this.props);
+  }
+
+  render() {
+    const movie = this.props.chosenMovie;
+    const getValues = movie.Ratings&&movie.Ratings.map((review) => <span key={review.Source} className="Reviews">{review.Source} : {review.Value}</span> );
```

Comment:
This is very long and hard to read. Consider using Prettier or something like it.
Also, using a linter would've helped you in general.